SwiftWisdom/Core/UIKit/UIView+Extensions.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  UIView+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Alexander Persian on 12/29/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+
+    /// Zooms a view by scaling using the provided factor.
+    func ip_zoom(by factor: CGFloat) {
+        transform = CGAffineTransform(scaleX: factor, y: factor)
+    }
+
+    /// Fades a view to alpha 0 over specified duration. Default = 0.2s
+    func ip_fadeOut(over duration: TimeInterval = 0.2) {
+        UIView.animate(
+            withDuration: duration,
+            animations: {
+                self.alpha = 0
+        },
+            completion: { [weak self] _ in
+                // Setting alpha to 1 here in case anyone tries to come back
+                // ... by just changing the `hidden` property.
```

Comment:
The method documentation should denote this behavior.  From reading the documentation I would expect the alpha to be 0 and my isHidden state to remain the same after this method is called.